app/Http/Controllers/DeviceGroupController.php

```diff
@@ -33,6 +35,8 @@ public function index()
         return view('device-group.index', [
             'device_groups' => DeviceGroup::orderBy('name')->withCount('devices')->get(),
             'ungrouped_devices' => $ungrouped_devices,
+            # FIXME: this shouldn't be needed here
```

Comment:
Will you fix these before merging?

Reply:
oh i oversees this,
with the "new" Gui i have to check this again, if it's still needed here.
Hopefully no :-)

Reply:
that's a bigger problem i think
this happens now on 

- global Settings
- Poller
- About
- (maybe more)

looks like this happens on laravel designed pages only.

i'll take a look over it,
if you have an idea where to define variable 'hide_dashboard_editor' global for all
Laravel based pages i will be happy to read it :-)

Reply:
found a fix